bazel/linux/defs.bzl

```diff
@@ -259,6 +259,16 @@ def _kernel_modules(ctx):
         modules = " ".join(modules),
     )
 
+    compilation_mode = ctx.var["COMPILATION_MODE"]
+    if compilation_mode == "fastbuild":
+        cflags = ""
+    elif compilation_mode == "opt":
+        cflags = ""
+    elif compilation_mode == "dbg":
+        cflags = "-O1 -fno-inline-functions-called-once"
+
+    extra += " BAZEL_CFLAGS+='%s'" % cflags
```

Comment:
just make this CFLAGS?

Reply:
Makes sense. Changed to `EXTRA_CFLAGS`.

Interesting trivia: make seems to be able to concatenate variables if asked to (with a +=) in the command line:
```
$ echo -e '$(info $(X))\nall:' | make -f - X=1 X+=2
1 2
```